README.md

```diff
@@ -366,6 +366,32 @@ const connector = new Connector({
 });
 ```
 
+This can be extremely useful to leverage the `Connector` with a service account that differs from that
```

Comment:
nit: extremely doesn't add anything to the sentence and it reads better without it

Reply:
This can be useful when configuring credentials that differ from Application Default Credentials.